### DISCUSSION TITLE
Issue running Motion

### DISCUSSION THREAD

**Discussion by PraiseUadiale** [Motion-Project/motion#1640] [Q&A]
Hi, 

Can someone help me understand what this error is?
For some context I am running Motion on a Raspi 3 using the Module 3 Camera

![image](https://user-images.githubusercontent.com/66537641/219089892-d4dac943-62e7-4ec8-989c-aeb0d20d1429.png)
![image](https://user-images.githubusercontent.com/66537641/219090069-cd5627d1-968a-4f57-8031-85a8004caab0.png)

**Comment by Mr-Dave** [comment]
It indicates `Device or resource busy` which means a different program is using the video device.  A search for this phrase will list many results to try.
